app/controllers/api/v1/users_controller.rb

```diff
@@ -25,6 +25,13 @@ def destroy
     render json: UserSerializer.new(User.destroy(params[:id]))
   end
 
+  def match_users 
+    if current_user 
```

Comment:
Makes sure that there is a logged in user and if there is it makes them match with users in the same location, ordered by their run pace.